StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -44,7 +44,7 @@ extension PaymentSheet {
         case dynamic(String)
         case UPI
         case cashApp
-
+        case externalPayPal // TODO(yuki): Replace this when we support more EPMs
```

Comment:
Would it be better to have a `.external(String)` case? I assume something like that would be the final shape of this, and I would prefer not to add a case that will be replaced later.

Reply:
It might be what we we end up with in the final version, but I don't want to preemptively do that now.  If I made that change, I'd have to deal with arbitrary strings wherever I use this type, but this implementation can't handle arbitrary external PMs - it only works for paypal.

Reply:
Fair